Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyAPI.Codeunit.al

```diff
@@ -317,31 +320,38 @@ codeunit 30286 "Shpfy Company API"
             Clear(ShopifyCompany.Note);
         ShopifyCompany.Modify();
 
+        IsDefaultCompanyLocation := true;
         if JsonHelper.GetJsonArray(JCompany, JLocations, 'locations.edges') then
```

Comment:
I know a lot of the codebase is very "jam everything in one procedure" but I think that we should make any place that we touch better than how we found it. And here, I think the location processing part should be extracted to it's own procedure. Maybe even further if you see fit...

Reply:
moved to separate procedure